src/main/scala/com/horizon/exchangeapi/BusinessRoutes.scala

```diff
@@ -379,6 +381,17 @@ trait BusinessRoutes extends JacksonSupport with AuthenticationSupport {
       ]
     }
   ],
+  "secretBinding": [
+     {
+       "serviceOrgid": "string",
+        "serviceUrl": "string",
+        "serviceArch": "amd64",
+        "serviceVersionRange": "x.y.z",
```

Comment:
Is a version range acceptable in the value of this field, e.g. "(1.0.0,2.0.0)" ?